src/openforms/js/components/admin/form_design/logic/actions/dmn/VariableMapping.js

```diff
@@ -85,7 +91,7 @@ const VariableMapping = ({
   alreadyMapped = [],
 }) => {
   const intl = useIntl();
-  const {getFieldProps, values} = useFormikContext();
+  const {values} = useFormikContext();
 
   const confirmationMessage = intl.formatMessage({
```

Comment:
Already there before your PR but if you change something else we can remove this I guess.